var/spack/repos/builtin/packages/gemma/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Gemma(MakefilePackage):
+    """GEMMA is a software toolkit for fast application of linear mixed models
+       (LMMs) and related models to genome-wide association studies (GWAS) and
+       other large-scale data sets."""
+
+    homepage = "https://github.com/genetics-statistics/GEMMA"
+    url      = "https://github.com/genetics-statistics/GEMMA/archive/refs/tags/v0.98.5.tar.gz"
+
+    maintainers = ['snehring', 'dlkuehn']
+
+    version('0.98.5', sha256='3ed336deee29e370f96ec8f1a240f7b62550e57dcd1694245ce7ec8f42241677')
+
+    depends_on('zlib')
+    depends_on('openblas')
```

Comment:
Only `openblas`? Or can we:
```suggestion
    depends_on('lapack')
```
?

Reply:
It has switches in the makefile to maybe make it work with atlas. Other blas/lapack implementations seem to result in linking errors due to specifying -latlas when openblas isn't specified.

Reply:
Thanks, can you either write that in a comment above the `depends_on` or see if we can make it work with other lapack implementations?

Reply:
The logic in the makefile is very simple, but it requires that the lapack implementor have literal clblas, lapack, and blas libraries, which many don't. netlib's libraries do, but mkl doesn't for instance.